scripts/genproto.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+#
+# Generate all etcd protobuf bindings.
+# Run from repository root directory named etcd.
+#
+set -e
+shopt -s globstar
+
+if ! [[ "$0" =~ scripts/genproto.sh ]]; then
+  echo "must be run from repository root"
+  exit 255
+fi
+
+source ./scripts/test_lib.sh
+
+if [[ $(protoc --version | cut -f2 -d' ') != "3.14.0" ]]; then
+  echo "could not find protoc 3.14.0, is it installed + in PATH?"
+  exit 255
+fi
+
+GOFAST_BIN=$(tool_get_bin github.com/gogo/protobuf/protoc-gen-gofast)
+GRPC_GATEWAY_BIN=$(tool_get_bin github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway)
+SWAGGER_BIN=$(tool_get_bin github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger)
+GOGOPROTO_ROOT="$(tool_pkg_dir github.com/gogo/protobuf/proto)/.."
+GRPC_GATEWAY_ROOT="$(tool_pkg_dir github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway)/.."
+
+echo
+echo "Resolved binary and packages versions:"
+echo "  - protoc-gen-gofast:       ${GOFAST_BIN}"
```

Comment:
I think raft does not expose `grpc` nor `json gateway`. 

Could you please check whether everything builds in case of: 

```
echo " - protoc-gen-gofast: ${GOFAST_BIN}"
echo " - gogoproto-root: ${GOGOPROTO_ROOT}"
```

Reply:
Confirmed that it's working, there was a minor syntax error in my previous experiment. Submitted another PR to cleanup it. https://github.com/etcd-io/raft/pull/6